examples/video-chat/screens/PreviewScreen/index.tsx

```diff
@@ -131,12 +133,20 @@ const PreviewScreen = ({ navigation, route }: Props) => {
     <SafeAreaView style={styles.container}>
       <View style={styles.cameraPreview}>
         {!isIosSimulator && isCameraOn ? (
-          <VideoPreviewView style={styles.cameraPreviewView} />
+          <VideoPreviewView
+            style={styles.cameraPreviewView}
+            mirrorVideo={mirrorPreview}
```

Comment:
User can mirror:
The reason why we mirror when swichig from front to back camera etc is because capturer gives mirrored preview. 
That is why i added is front and why there is xor. Few comments before i explained it. And now in our code we only modify `isFrontCammera`. Mirror property stays the same on it can be only modified by user